src/main/java/core/basesyntax/service/impl/DeepCopyImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitTransaction;
+import core.basesyntax.service.DeepCopy;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DeepCopyImpl implements DeepCopy {
```

Comment:
The class name `DeepCopyImpl` suggests that it is a generic implementation for deep copying, but it specifically deals with `FruitTransaction` objects. Consider renaming the class to reflect its specificity, such as `FruitTransactionDeepCopyImpl`.